### DISCUSSION TITLE
Share cloudflare cache rules

### DISCUSSION THREAD

**Discussion by Justinidlerz** [esm-dev/esm.sh#518] [General] [answered]
Hi @ije, we just self-hosted the ESM server, but without CDN cache rules, could you please share it with us?
And can we have a DM channel? Maybe WeChat or Slack or any others. thanks

**Comment by ije** [comment, marked as answer]
esm.sh uses CF's workers as the front layer, i added only one page rule to allow **Cache Everything**, then handle the cache in workers script, basically following the origin server `cache-control` header

**Reply by ije** [reply]
**fyi** CF's CDN only cache specified types by checking the extname in uri, not respect the `content-type`

**Comment by ije** [comment]
you can DM me by discord `X.#9041`
